### PR TITLE
[BEAM-947] Fix breakage in WindowedWordCountIT

### PR DESCRIPTION
**Pull request by kennknowles**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

R: anyone
CC: @jasonkuster @davorbonaci

### CODE REVIEW

**Comment by jasonkuster** [comment]
LGTM

**Comment by kennknowles** [comment]
Failed in DataflowRunner without an error message being returned; got past the initial options breakage that this PR addresses. We'll see if it is repeatable.

**Comment by kennknowles** [comment]
Dataflow service was happy this time, but file checksum failed, likely due to [BEAM-747](https://issues.apache.org/jira/browse/BEAM-747). Kicking tests.

File: examples/java/src/test/java/org/apache/beam/examples/WindowedWordCountIT.java [hunk 1]
```diff
@@ -67,6 +68,14 @@ private void testWindowedWordCountPipeline(boolean isStreaming) throws IOExcepti
         TestPipeline.testingPipelineOptions().as(WindowedWordCountITOptions.class);
     options.setStreaming(isStreaming);
 
+    // Note: currently unused because the example writes to BigQuery, but WindowedWordCount.Options
+    // are tightly coupled to WordCount.Options, where the option is required.
+    options.setOutput(IOChannelUtils.resolve(
```
Comment (davorbonaci) [inline examples/java/src/test/java/org/apache/beam/examples/WindowedWordCountIT.java]: Perhaps not at the right level of abstraction. I'd prefer if test infrastructure (TestPipeline, TestPipelineOptions, TestRunner, etc.) could hold this functionality, and the test calls it to get output location.

Comment (kennknowles) [inline examples/java/src/test/java/org/apache/beam/examples/WindowedWordCountIT.java]: I don't have a strong opinion at this time. This replicates the current practice from `WordCountIT`:

https://github.com/apache/incubator-beam/blob/master/examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java#L59

**Comment by kennknowles** [comment]
@davorbonaci PTAL. Made hardcoded (!!) timeout configurable and increased it so tests could pass.

**Comment by davorbonaci** [comment]
I'd prefer if the test timeout is per-test, as opposed to the per-suite.

**Comment by kennknowles** [comment]
Made the timeout per-test.

**Comment by davorbonaci** [comment]
LGTM

**Comment by kennknowles** [comment]
Will merge when Jenkins is green.

**Comment by kennknowles** [comment]
Failure was [BEAM-957](https://issues.apache.org/jira/browse/BEAM-957)

**Pull request closed without merging**